source/ShipInfoPanel.cpp

```diff
@@ -230,11 +234,20 @@ bool ShipInfoPanel::Click(int x, int y, int /* clicks */)
 
 	draggingIndex = -1;
 	if(panelState.CanEdit() && hoverIndex >= 0 && (**shipIt).GetSystem() == player.GetSystem() && !(**shipIt).IsDisabled())
+	{
 		draggingIndex = hoverIndex;
+		dragRight = hoverRight;
+	}
 
 	selectedCommodity.clear();
 	selectedPlunder = nullptr;
 	Point point(x, y);
+	if(point.X() <= nextHardpoint.X() + 145. && point.X() >= nextHardpoint.X() + 105.
```

Comment:
These values should probably be in interface.txt, it's not really important if the rest cannot be customized (havent looked if thats the case), at least we shouldnt add fixed elements

Reply:
Actually after having a deep look, this element is positioned according to how the hardpoints are displayed, which makes it not fixed

Reply:
The relative distance is though

Reply:
how can I set this distance in interfaces?
I am not using interfaces.txt very often...

Reply:
just look at the mapplanetcard.cpp it uses this extensively